packages/fluentui/accessibility/src/behaviors/Tree/treeTitleAsListItemTitleBehavior.ts

```diff
@@ -14,7 +17,8 @@ export const treeTitleAsListItemTitleBehavior: Accessibility<TreeTitleBehaviorPr
     attributes: {
       root: {
         ...(!props.hasSubtree && {
-          role: 'listitem',
+          role: props.selectable ? 'option' : 'listitem',
```

Comment:
very similar issue as with "aria-selected"
because we need to be consistent and has on whole selectable list just role=option, not mix there 'listitem'
![image](https://user-images.githubusercontent.com/22620150/93877585-f44c5480-fcd8-11ea-8a3f-e3a9eba60801.png)

Reply:
So if a parent div is `listbox`, all the children should be `option` even if it is not selectable?